### ISSUE TITLE
Python thread never stops

### ISSUE THREAD

**Issue by lgrahl** [lgrahl/scripthookvpy3k#6]
This causes two issues:
- The ScriptHook can't detach (and reattach) the ASI plugin
- The GTA V process does not stop after exiting the game and needs to be killed by hand

Potentially influences other ASI plugins as well (untested).

**Comment by lgrahl** [comment]
Below is the error that is raised when trying to execute [this function](https://github.com/lgrahl/scripthookvpy3k/blob/3f6d9bc2354678f653450e3307c2a3ebdb4f0bf5/python/gta/__init__.py#L118) after `DLL_PROCESS_DETACH`:

```
2015-05-16 18:56:44 asyncio                DEBUG              Fail to write a null byte into the self-pipe socket
Traceback (most recent call last):
  File "C:\Python34\Lib\asyncio\selector_events.py", line 146, in _write_to_self
    csock.send(b'\0')
OSError: [WinError 10093] Either the application has not called WSAStartup or WSAStartup failed
```

It works absolutely fine when I reinitialise Python (by pressing F12). I've tried the `ProactorEventLoop` as well which doesn't raise this error but doesn't run the callback `__stop` either.

**Comment by lgrahl** [comment]
The GTA V process seems to stop now. Closed and partly moved to #10.
